frameworks/TypeScript/elysia/src/postgres.ts

```diff
@@ -9,16 +9,16 @@ const sql = postgres({
   max: 1,
 });
```

Comment:
why `max: 1`? is it rule or some kind of performance hack?

Reply:
I can't speak to a performance hack, but the number of max connections is not a rule.